packages/utils/test-tools/src/mockTheme.ts

```diff
@@ -175,6 +175,18 @@ export const mockTheme: Theme = {
       heroLargeSemibold: { face: 'primary', size: 'heroLarge', weight: 'semiBold' },
     },
   },
+  effects: {
+    borderRadiusSmall: 2,
+    borderRadiusMedium: 4,
+    borderRadiusLarge: 6,
+    borderRadiusXLarge: 8,
+    shadow2: { ambient: undefined, key: undefined },
+    shadow4: { ambient: undefined, key: undefined },
+    shadow8: { ambient: undefined, key: undefined },
+    shadow16: { ambient: undefined, key: undefined },
+    shadow28: { ambient: undefined, key: undefined },
+    shadow64: { ambient: undefined, key: undefined },
```

Comment:
Why are all these undefined?

Reply:
I was being lazy for the hackathon and I'm not sure it matters terribly much for the test this mock is used in